core/lib/spree/testing_support/factories/order_factory.rb

```diff
@@ -5,6 +5,7 @@
     completed_at nil
     email { user.email }
     store
+    currency Spree::Config[:currency]
```

Comment:
@VygovskySergey let's change it to just `USD`, testing environment shouldn't be dependant on preferences